Clinical-Genomics/cg
# support for OF 1508:15

This PR adds support for OF 1508:15

How to test:
0. install on stage clinical-api & cgweb 
1. Import 1508:15 OF

Expected outcome:
It imports w/o problems
